packages/framework/presence/src/presenceDatastoreManager.ts

```diff
@@ -73,7 +73,7 @@ const joinMessageType = "Pres:ClientJoin";
 interface ClientJoinMessage extends IInboundSignalMessage {
 	type: typeof joinMessageType;
 	content: {
-		updateProviders: ClientConnectionId[];
+		updateProviders?: ClientConnectionId[];
```

Comment:
It is not safe to just remove items from message protocol. We don't have anything in use right now that we know of, but we'll want to be careful about changes. Here it would be fine to specify updateProviders even if targeting is supported. If there was older version of presence connected to session they would be expecting updateProviders to be defined; so, we should keep it defined.